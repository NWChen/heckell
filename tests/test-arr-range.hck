let x: int array;
x = [1, 3 ... 7];

print("\(x.get(0))");
print("\(x.get(1))");
print("\(x.get(2))");
print("\(x.get(3))");