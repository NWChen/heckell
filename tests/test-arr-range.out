1
3
5
7
19
20
